.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,7 +10,7 @@ Thanks for stopping by to let us know something could be better!
 
 Please run down the following list and make sure you've tried the usual "quick fixes":
 
-  - Search the issues already opened: https://github.com/googleapis/java-bigtable/issues
+  - Search the issues already opened: https://github.com/googleapis/google-cloud-bigtable/issues
```

Comment:
I think this is wrong

Reply:
This is generated by synthtool based off of .repo-metadata.json. If this is incorrect let's raise an issue at the template level